src/classes/CRLP_Rollup_SEL.cls

```diff
@@ -62,6 +63,16 @@ public class CRLP_Rollup_SEL {
                         WHERE Is_Deleted__c = false
                         ORDER BY Summary_Object__c, MasterLabel
                 ];
+
+                // Never return Rollups with banned Target Fields
+                // You can't create them in the CRLP UI, but they may have snuck in via native CMDT UI
+                for (Rollup__mdt rollup : stagedRollups) {
```

Comment:
I understand what you're going for here, but I'm wondering if this is overkill. I feel like we're trying to prevent users from harming themselves, which to me goes outside of the scope of this project and our role overall. We're giving them a tool to use. If they purposely choose to use Salesforce Settings or a metadata deployment tool to insert CMT's that our UI wouldn't normally support that is their choice. I don't feel like we have the right to tell them they can't do it. @njjc What do you think?

Reply:
I agree with Mike..."we don't recommend you shoot yourself in the foot, but we're not going to provide you with bullet-proof shoes to help avoid it."

Reply:
OK, I'll take it out. Thanks!